README.md

```diff
@@ -238,6 +238,15 @@ a.b = 1
 a.b.c = 2
 ```
 
+Defining dotted keys out-of-order is valid but discouraged:
+
+```
+o.1.1 = 1.1
+o.2.1 = 2.1
+o.1.2 = 1.2
+o.2.2 = 2.2
```

Comment:
This IS totally valid `.toml`, but is a little odd because it goes against the admonishment to not do things that look like "float keys" (multiple keys by integers). Then again it's an example for "what not to do, unless you're really sure", so... appropriate?